tests/multimodal/test_processor.py

```diff
@@ -12,7 +12,7 @@
 @pytest.mark.parametrize("dtype", ["half", "float"])
 def test_clip_image_processor(hf_images, dtype):
     MODEL_NAME = "llava-hf/llava-1.5-7b-hf"
-    IMAGE_HEIGHT = IMAGE_WIDTH = 33
+    IMAGE_HEIGHT = IMAGE_WIDTH = 560
```

Comment:
I think I originally made a typo (meant to be 336 instead of 33). But I should have made the image larger than that anyway to test whether HF does resizing in the same way.